### PR TITLE
Make largeint() generate integers

### PR DESCRIPTION
**Pull request by nickva**
Previously it generated floats.

### CODE REVIEW

File: src/triq_dom.erl [hunk 1]
```diff
@@ -655,21 +655,18 @@ pos_integer() ->
...
-                      }
+                     Val = trunc(?BIGNUM * SampleSize * triq_rnd:uniform()),
+                     Sign = case triq_rnd:uniform(2) of
+                         2 ->
+                             -1;
+                         1 ->
+                             1
+                     end,
+                     {Dom, Sign * Val}
              end
```
Comment (tuncer) [inline src/triq_dom.erl]: Minor style nit: this doesn't seem to be indented by erlang.el's formatter.

Comment (nickva) [reply src/triq_dom.erl]: Thanks. Will fix

File: src/triq_dom.erl [hunk 2]
```diff
@@ -655,21 +655,18 @@ pos_integer() ->
...
-                      ?BIGNUM
-                      * SampleSize * triq_rnd:uniform()
-                      * case triq_rnd:uniform(2) of
-                            2 ->
-                                -1;
-                            1 ->
-                                1
-                        end
-                      }
+                     Val = trunc(?BIGNUM * SampleSize * triq_rnd:uniform()),
```
Comment (KrzysiekJ) [inline src/triq_dom.erl]: Since `?BIGNUM` is supposed to be an integer, why do we need `trunc/1` here?

Comment (nickva) [reply src/triq_dom.erl]: `triq_rnd:uniform()` produces floats so then the whole expression is cast to a float

Comment (KrzysiekJ) [reply src/triq_dom.erl]: Ahh, `triq_rnd:uniform/0` produces floats and `triq_rnd:uniform/1` produces ints (analogous to the `rand` module). Ok.

Comment (tuncer) [reply src/triq_dom.erl]: We can get an `integer()` if we call `uniform(N)` where N >= 1.

Comment (tuncer) [reply src/triq_dom.erl]: Yeah, if we call `uniform(N)` where N >= 1, then we will get an `integer()` (same as directly calling that function from `rand` or `random`).

**Pull request merged**